servers/quarkus-server/src/intTest/java/org/projectnessie/server/catalog/s3/ITTrinoS3.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2024 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server.catalog.s3;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import java.net.URI;
+import java.util.Map;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.nessie.testing.trino.TrinoContainer;
+import org.projectnessie.nessie.testing.trino.TrinoResults;
+import org.projectnessie.server.catalog.MinioTestResourceLifecycleManager;
+import org.projectnessie.server.catalog.WarehouseLocation;
+import org.testcontainers.Testcontainers;
+
+@QuarkusTestResource(
+    restrictToAnnotatedClass = true,
+    value = MinioTestResourceLifecycleManager.class)
+@QuarkusIntegrationTest
+public class ITTrinoS3 {
+  static TrinoContainer trinoContainer;
+
+  @WarehouseLocation static URI warehouseLocation;
+
+  static String icebergRestUri;
+
+  @BeforeEach
+  void maybeStartTrino() {
```

Comment:
Nit: could be turned into `@BeforeAll`.

Reply:
Nope - the s3 location wouldn't be available